index.js

```diff
@@ -328,6 +330,8 @@ module.exports = {
     'max-depth': 1,
     // specify the maximum length of a line in your program
     'max-len': [1, 80, 2],
+    // enforce a maximum file length
+    'max-lines': 0,
```

Comment:
It means this rule is disabled.  For all of the rules, a single value means 'off' (0), 'warning' (1), or 'error' (2).  If there is configuration, then it goes in an array, where the first element is the same as above, and the remaining values are config.

ESLint recently introduced support for using words instead of numbers; might be a good idea to adopt that at some point to make the config clearer.